script/Ciel/Day 9/09_01_CIEL09_8.txt

```diff
@@ -11,32 +11,40 @@
 [sha:e60c1c7bc496fcdd8ac3b3f6b5e1281aca23c46e]{
 -- Page 10027, Offset 23940.
 -- 一瞬の出来事だった。
-It happens in the blink of an eye.
+It happened in the blink of an eye.
 }
 [sha:c359770e05f8c7037acc4c9e02a82de635900828]{
 -- Page 10027, Offset 23941.
 -- 満足に立ち上がれないほど傷ついていた先輩が、俺の前に回りこんできた。
-Senpai, who is so injured she can't even properly stand up, appears before me.
+Senpai was so injured she couldn't even stand a second ago,
+ but now she's standing between Arcueid and me.
```

Comment:
I prefer the but to the clause, it's a snappier sentence. Changed stand.

Reply:
Hmmm, there's just something about the pace of the sentence that sounds very off to me.
At the very least 'Arcueid and me' sounds like it should be 'me and Arcueid'

Reply:
Reordered to avoid both and and the second clause. Arcueid and me is correct, I don't believe you ever start an X and I / X and me with oneself. 
> Despite being so injured she couldn't even stand a second ago,
>  Senpai has placed herself between Arcueid and me.